### PR TITLE
Remove ES6 Exporter and JSON Schema Exporter

### PR DESCRIPTION
**Pull request by jafeltra**
This PR removes the `shr-es6-exporter` and `shr-json-schema-exporter`. I also removed parts of the code in other packages that seemed relevant to these exporters. It is very possible that I missed pieces of code that were added to other packages specifically for either of these exporters. If anyone knows of other code that can be removed, let me know.

Bonus for Mac users: I also added .DS_Store to the gitignore.

### CODE REVIEW

File: packages/shr-fhir-export/lib/extensions.js
Comment (kjmahalingam) [inline packages/shr-fhir-export/lib/extensions.js:101]: Nice

File: packages/shr-cli/app.js
Comment (kjmahalingam) [inline packages/shr-cli/app.js]: Now that we have two separate conditionals with the same condition, could you put this statement inside the conditional at line 187?

**Comment by kjmahalingam** [comment]
Assuming the above minor comment is addressed, this is good to me! I tested it and it works as expected as well. Great work!

@cmoesel since this is such a drastic change, I'm going to leave it to you to decide when it should be actually merged.

File: packages/shr-fhir-export/lib/extensions.js
Comment (jafeltra) [reply packages/shr-fhir-export/lib/extensions.js:101]: It took removing and then replacing that line for me to make this monumental change.

File: packages/shr-cli/app.js
Comment (jafeltra) [reply packages/shr-cli/app.js]: Oh yes. I didn't even notice that. Good catch. I'll make that change.

**Comment by jafeltra** [comment]
@kjmahalingam your comment should be addressed. I moved the line in the first conditional into the try block of the second conditional.

**Review by kjmahalingam — APPROVED** [review]
Good work! @cmoesel this is good for merge whenever you're comfortable doing so.

File: packages/shr-expand/README.md
Comment (cmoesel) [inline packages/shr-expand/README.md:18]: Oh wow.  This line of text comes from a long time ago, like the first time we had all the code together in a single repo.  It's not quite right, but... I won't hold up the PR for this little nugget.

**Pull request merged**
